tests/scorers/test_bias_scorer.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+
+from tests.scorers.test_utils import TINY_MODEL_PATHS, generate_large_text
+from weave.scorers.llm_utils import download_model
+from weave.scorers.moderation_scorer import BiasScorer
+
+
+@pytest.fixture
+def bias_scorer():
+    """Fixture that returns a BiasScorer instance using a tiny downloaded model."""
+    tiny_model_path = download_model(TINY_MODEL_PATHS["bias_scorer"])
+    return BiasScorer(
+        model_name_or_path=tiny_model_path,
+        name="test-bias",
+        description="Tiny test bias scorer",
+        device="cpu",  # Use CPU for testing
+        threshold=0.5,
+    )
+
+
+def test_bias_scorer_simple(bias_scorer):
+    """Tests that the scorer can handle a basic string."""
+    output = "This is a balanced statement with no bias."
+    result = bias_scorer.score(output)
+
+    assert isinstance(result, dict), "Result should be a dictionary."
+    assert "flagged" in result, "Result must contain the 'flagged' key."
+    assert "extras" in result, "Result must contain the 'extras' dictionary."
+    assert isinstance(result["flagged"], bool), "'flagged' must be a boolean."
+    assert isinstance(result["extras"], dict), "'extras' must be a dictionary."
+    for category in bias_scorer._categories:
+        assert (
+            category in result["extras"]
+        ), f"'{category}' should be in 'extras' dictionary."
```

Comment:
Instead of doing this, just make a `TypedDict` and return it on the scorer.  Then you don't need these